charts/lerian-consul/templates/validate-quorum.yaml

```
@@ -0,0 +1,17 @@
+{{- /*
+Render-time guardrail for the Consul Raft quorum. The upstream chart renders a
+broken cluster without complaint: bootstrapExpect != replicas never forms
+quorum, and an even replica count tolerates zero failures while risking
+split-brain. Fail fast with a clear message instead. Emits nothing on success.
+*/ -}}
+{{- $server := (.Values.consul | default dict).server | default dict -}}
+{{- $replicas := int ($server.replicas | default 1) -}}
+{{- if $server.bootstrapExpect -}}
+{{-   $expect := int $server.bootstrapExpect -}}
+{{-   if ne $replicas $expect -}}
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)